### ISSUE TITLE
Trigger apply run from "digger apply" on PR review comments

### ISSUE THREAD

**Issue by noralutz** [diggerhq/digger#1220]
Currently, this comment does not cause Digger to react, meaning I have to make another comment after I approve.

**Comment by motatoes** [comment]
Interesting, looks like that triggers a `pull_request_review_comment` event :D . We need to handle that in orchestrator and handle it as a comment event 👍

**Comment by jwtly10** [comment]
Hey @motatoes @noralutz, I've reproduced this, so perhaps I can take a look into making a fix.

**Comment by ahuseby** [comment]
This would be a good addition indeed.

An important difference between `issue_comment` and `pull_request_review_comment` is that `GITHUB_REF` is the default branch for the former, but `refs/pull/PULL_REQUEST_NUMBER/merge` for the latter (see [docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review_comment)). This makes the review comment more integrated into the pull request context (e.g. not having to do [this](https://github.com/diggerhq/digger/blob/6bbd1e0e7452091a469c9b9a123ac3d6d34b87dd/action.yml#L164-L168)).

A GitHub [feature request](https://github.com/orgs/community/discussions/59389) has been raised for making `issue_comment` in PRs (or a new PR comment event type) use the PR ref, but until then the `pull_request_review_comment` is the best option.

**Comment by motatoes** [comment]
Thanks @ahuseby ! So we will need to that event and handle it on digger side. I think issue number will still remain so that action.yml logic would still work

**Comment by ahuseby** [comment]
> Thanks @ahuseby ! So we will need to that event and handle it on digger side. I think issue number will still remain so that action.yml logic would still work

That's right 😄 I'll try and take a look at contributing a change for this
